src/gossip/service.zig

```diff
@@ -3436,3 +3438,21 @@ fn localhostTestContactInfo(id: Pubkey) !ContactInfo {
     try contact_info.setSocket(.gossip, SocketAddr.initIpv4(.{ 127, 0, 0, 1 }, 0));
     return contact_info;
 }
+
+// test "benchmarkPullRequests" {
+//     _ = try BenchmarkGossipServicePullRequests.benchmarkPullRequests(.{
+//         .name = "1k_data_1k_pull_reqs",
+//         .n_data_populated = 10,
+//         .n_pull_requests = 2,
+//     });
+// }
+
+// test "benchmarkGossipService" {
+//     _ = try BenchmarkGossipServiceGeneral.benchmarkGossipService(.{
+//         .message_counts = .{
+//             .n_ping = 10,
+//             .n_push_message = 10,
+//             .n_pull_response = 10,
+//         },
+//     });
+// }
```

Comment:
Are we fixing and re-enabling these, or removing these?

Reply:
left a comment - right now they leak when run with the testing allocator -- will fix the leak in another PR since its a bit more involved to fix

[bb8257f](https://github.com/Syndica/sig/pull/316/commits/bb8257f935ab417799c53a90ccdd3aa96747ce15)

https://github.com/orgs/Syndica/projects/2/views/10?pane=issue&itemId=84676970